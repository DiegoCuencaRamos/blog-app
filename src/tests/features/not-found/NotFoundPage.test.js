import React from 'react'
import { shallow } from 'enzyme'
import NotFoundPage from '../../../features/NotFoundPage'

test('should render NotFoundPage', () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(wrapper).toMatchSnapshot();
})