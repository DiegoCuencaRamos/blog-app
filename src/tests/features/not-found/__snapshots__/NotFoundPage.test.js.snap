// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage 1`] = `
<div>
  Not found page.
</div>
`;
